dataland-frontend/src/components/general/CompanyDataTable.vue

```diff
@@ -54,7 +54,7 @@
 
       <Column field="subAreaKey" header="Impact Area"></Column>
       <template #groupheader="slotProps">
-        <span>{{
+        <span :id="slotProps.data.subAreaKey" :data-row-header-click="slotProps.data.subAreaKey">{{
```

Comment:
Why is the same value assigned to two different html attributes?
It should be possible to do this to only one and use this one everywhere.

Reply:
To separate functionalities so that features can work independently. If you feel that's overkill then we can operate solely on `id`

Reply:
Ok I understand. Good point.
My only suggestion then is to set the value of "data-row-header-click" to a static value, because it does not really matter what value it contains when it is used as selector in `mountRowHeaderClickEventListeners()`.  This should then also be done in the `TwoLayerDataTable.vue`.

Reply:
Changed to just setting the attribute without value ... `data-row-header-click`